cea/datamanagement/zone_helper.py

```diff
@@ -72,12 +72,6 @@ def assign_attributes(shapefile, buildings_height, buildings_floors, buildings_h
               'if we do not find data in OSM for a particular building, we estimate it based on the number of floors, '
               'multiplied by a pre-defined floor-to-floor height (set to 3m by default)')
 
-        # Make sure relevant OSM parameters (if available) are passed as floats, not strings
-        OSM_COLUMNS = ['building:min_level', 'min_height', 'building:levels', 'height']
-        selected_columns = list(set(list_of_columns).intersection(set(OSM_COLUMNS)))
-        shapefile[selected_columns] = shapefile[selected_columns] \
-            .fillna(1).apply(lambda x: pd.to_numeric(x, errors='coerce'))
-
```

Comment:
@MatNif 
Correct me if I am wrong. The original Line 75-80 and Line 81-85 are duplicated. Hence, I remove the Line75-80

Reply:
Sorry, I think you're right. I missed that.

In that case, I think this PR is good to go.